Wikipedia/View Controllers/Nearby/NearbyViewController.m

```diff
@@ -122,12 +113,50 @@ - (BOOL)prefersStatusBarHidden {
     return NO;
 }
 
-- (void)viewWillDisappear:(BOOL)animated {
-    [self.locationManager stopUpdatingLocation];
+- (void)viewWillAppear:(BOOL)animated {
+    [super viewWillAppear:animated];
+
+    self.skipStartUpdates = NO;
+    // Only begin updates if by the time the startUpdates selector below is performed
+    // self.skipStartUpdates is still NO - lets us add the nearby view controller
+    // somewhere and immediately cover it up without it starting to update locations
+    // and headings.
+    [self performSelector:@selector(startUpdates) withObject:nil afterDelay:0.5];
```

Comment:
makes sense - at some point we will have some UI to allow users to opt in - just not yet